src/app/components/delegate-registration/delegateRegistration.pug

```diff
@@ -0,0 +1,23 @@
+md-dialog.dialog-delegate-registration(aria-label='Transfering', style='width: 700px; max-width: 700px')
+  md-card
+    md-card-title
+      md-card-title-text
+        span.md-title Send
+    md-card-content
+      form(name='delegateRegistrationForm', ng-submit='$ctrl.form.onSubmit(delegateRegistrationForm)')
+        md-content(flex='100', flex-gt-sm='70', flex-offset-gt-sm='15')
+          md-input-container.md-block
+            label Delegate name
+            input(type='text', name='delegateName', ng-model='$ctrl.form.name', required, ng-disabled='$ctrl.loading')
+            div(ng-messages='delegateRegistrationForm.name.$error')
+              div(ng-message='required') Required
+              //- div(ng-message='pattern') Invalid
+            md-input-container.md-block
+              div.fee Fee: {{$ctrl.form.fee}} LSK
+            md-input-container.md-block
+              p Becoming a delegate requires registration. You may choose your own delegate name, which can be used to promote your delegate. Only the top 101 delegates are eligible to forge. All fees are shared equally between the top 101 delegates.
```

Comment:
I would make this look more like the info block in vote dialog [0], that is: use "info" icon and separate by md-divider:
[0] https://github.com/LiskHQ/lisk-nano/blob/development/src/app/components/delegates/vote.pug#L22-L30